src/main/data/PlaceField.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * The Place Data Fields from https://developers.google.com/places/web-service/place-data-fields.
+ * Specifies the types of Place data to return when requesting Place Details
+ * */
+export enum PlaceField {
```

Comment:
Do we need all these fields?

Reply:
Right now, no. But in the future maybe, I think there's no harm having the complete set of fields here. Is that okay?